ComposureWatch/src/Components/Sorting/SortingLogic.jsx

```diff
@@ -1,5 +1,7 @@
 // To be determine if ranks will be part of the rank icon list or housed seperately here for values
 import CardList from "../Card/CardList";
+import TeamOneCardList from "../Card/TeamOneCardList";
+import TeamTwoCardList from "../Card/TeamTwoCardList";
```

Comment:
Do you need two components in this case? I think the only difference I saw was the border & background color ... maybe you could pass those in as props or even a team value and you set them in the component?

I feel like if you ever wanted to change the TeamOne component styling/structure you would want to also change the TeamTwo component ... which makes me think these are sorta one in the same.

```
<TeamList cards={teamOne} team={1} />
<TeamList cards={teamTwo} team={2} />

// or
<TeamList cards={teamOne} colors={{ bg: "#0A0D30", border: "#2EF3FF" }} />
// or
<TeamList cards={teamOne} bgColor="" borderColor="" />
```

Have some fun with it 😄